charts/ingress-nginx/values.yaml

```diff
@@ -212,6 +212,10 @@ controller:
     timeoutSeconds: 1
     port: 10254
 
+  # Path of the health check endpoint. All requests received on the port defined by
+  # the healthz-port parameter are forwarded internally to this path.
```

Comment:
@aledbf both `controller.healthCheckPath` documentation in `charts/ingress-nginx/README.md` and here reference `healthz-port` parameter. I can see `controller.readinessProbe.port` and `controller.livenessProbe.port`, but I cannot find one named `healthz-port`. Can you please clarify, is `healthz-port` reference just a generic term for the other two, or is `healthz-port` actual parameter somewhere?

Reply:
@sslavic are deployment flags https://kubernetes.github.io/ingress-nginx/user-guide/cli-arguments/
--health-check-path 
--healthz-port